wolfcrypt/test/test.c

```diff
@@ -122,7 +122,8 @@
 
 
 #if defined(USE_CERT_BUFFERS_1024) || defined(USE_CERT_BUFFERS_2048) \
-                                   || !defined(NO_DH)
+                                   || !defined(NO_DH) \
+                                   || defined(USE_CERT_BUFFERS_256)
     /* include test cert and key buffers for use with NO_FILESYSTEM */
```

Comment:
I believe "USE_CERT_BUFFERS_256" can be used even with NO_DH defined. If in-fact the USE_CERT_BUFFERS_1024 and USE_CERT_BUFFERS_2048 shouldn't be included with NO_DH then consider moving that check into the certs_test.h. Then include certs_test.h all the time here.